packages/cli/src/decorators/Scopes.ts

```diff
@@ -0,0 +1,17 @@
+import type { Scope, ScopeOptions } from '@n8n/permissions';
+import type { ScopeMetadata } from './types';
+import { CONTROLLER_REQUIRED_SCOPES } from './constants';
+
+export const RequireGlobalScope = (scope: Scope | Scope[], scopeOptions?: ScopeOptions) => {
```

Comment:
`scopeOptions` (and `options` in `ScopeWithOptions`) doesn't seem to be used. should we remove it for now, and add it when we actually need it ?